service/src/test/java/org/sunbird/learner/actors/skill/UserSkillManagementActorTest.java

```diff
@@ -39,39 +36,39 @@
 
   private static ActorSystem system;
   private static final Props props = Props.create(UserSkillManagementActor.class);
-  private static CassandraOperation cassandraOperation;
-  private static final String USER_ID = "someUserId";
+  private CassandraOperation cassandraOperation;
+  private static final String USER_ID = "userId";
   private static final String ROOT_ORG_ID = "someRootOrgId";
   private static final String ENDORSED_USER_ID = "someEndorsedUserId";
-  private static final String ENDORSE_SKILL_NAME = "C";
-  private static List<String> skillsList = new ArrayList<>();
-  private static FiniteDuration duration = duration("10 second");
+  private static final String ENDORSED_SKILL_NAME = "someEndorsedSkillName";
+  private FiniteDuration duration = duration("10 second");
 
   @BeforeClass
-  public static void setUp() {
-    PowerMockito.mockStatic(ElasticSearchUtil.class);
+  public static void setup() {
     system = ActorSystem.create("system");
+    PowerMockito.mockStatic(ServiceFactory.class);
```

Comment:
all static c=mocks can be added here- 
rename to setUp - codacy detects this as an issue